web_monitoring/tests/test_diffing_server_exc_handling.py

```diff
@@ -0,0 +1,55 @@
+from tornado.testing import AsyncHTTPTestCase
+import web_monitoring.diffing_server as df
+from web_monitoring.diff_errors import UndecodableContentError
+
+class DiffingServerExceptionHandlingTest(AsyncHTTPTestCase):
+
+    def get_app(self):
+        app = df.make_app()
+        app.listen(str(self.get_http_port()))
+        return
+
+    def testInvalidURLaFormat(self):
```

Comment:
Please use underscore case for method names: `def test_invalid_url_a_format`. Same for the rest of these tests.